README.md

```diff
@@ -97,7 +97,7 @@ license.
 [issues]: https://github.com/mixxxdj/mixxx/issues
 [fileabug]: https://github.com/mixxxdj/mixxx/issues/new/choose
 [mastodon]: https://floss.social/@mixxx
-[twitter]: https://twitter.com/mixxxdj
+[Bluesky]: https://bsky.app/profile/mixxx.bsky.social
```

Comment:
so we don't forget later, we're also trying to make that @mixxx.org domain thing (which is apparently possible with bsky). so that'll have to get updated then later.